server/opts.go

```diff
@@ -705,15 +705,17 @@ func (o *Options) processConfigFileLine(k string, v interface{}, errors *[]error
 			}
 			o.TrustedOperators = append(o.TrustedOperators, opc)
 		}
-		// In case "resolver" is defined as well, it takes precedence
-		if o.AccountResolver == nil && len(o.TrustedOperators) == 1 {
-			if accUrl, err := parseURL(o.TrustedOperators[0].AccountServerURL, "account resolver"); err == nil {
-				// accommodate nsc which appends "/accounts" during nsc push
-				suffix := ""
-				if accUrl.Path == "/jwt/v1/" || accUrl.Path == "/jwt/v1" {
-					suffix = "/accounts"
```

Comment:
This gets removed. Could you explain why? You say "nsc automatically append "/accounts" during nsc push, but the original comment says "accommodate nsc which appends "/accounts" during nsc push" and yet checks path and adds suffix.

Reply:
We did this optionally to do what nsc does during push. (only place where the account server url is used)
Alberto was unhappy conditionally appending /accounts and wanted it to happen all the time.
That way switching t a /jwt/v2 won't break this and if in the future we need to resolve different types of jwt, we can...